drivers/bluetooth/hci/spi.c

```diff
@@ -285,17 +285,72 @@ static int bt_spi_send_aci_config_data_controller_mode(void)
 }
 #endif /* CONFIG_BT_BLUENRG_ACI */
 
-static void bt_spi_rx_thread(void)
+static struct net_buf *bt_spi_rx_buf_construct(uint8_t *rxmsg)
```

Comment:
Not completely sure about the "construct" name, since it doesn't only do that: it actually consumes the data in the case of a vendor packet.

Reply:
Sort of true, the only event `bt_spi_handle_vendor_evt` ever actually consumes is `EVT_BLUE_INITIALIZED`, everything else is passed up the stack. Originally I called it `handle`, but its not really doing that either, just constructing the `net_buf`. Happy to hear any other name suggestions.